Improve template documentation for mint new (#5270)
mintlify/docs: cli/install.mdx

| `--template` | Pre-defined template. The CLI prompts for this if not provided. |
| `--force` | Overwrite the current directory without prompting. |

In interactive mode, the CLI asks whether you want to pick a theme or clone a template. To skip the prompt, pass the `--template` flag directly:

```bash
mint new my-docs --template <template-name>
```

You can combine `--template` with `--theme` to override the template's default theme:

```bash
mint new my-docs --template <template-name> --theme <theme>
```

> Templates are sourced from the [mintlify/templates](https://github.com/mintlify/templates) repository on GitHub. In interactive mode, the CLI fetches and displays available templates automatically.

Reviewer: ```suggestion
See available templates in the [mintlify/templates](https://github.com/mintlify/templates) repository on GitHub. In interactive mode, the CLI fetches and displays available templates automatically.
```
